packages/block-library/src/cover/edit.native.js

```diff
@@ -225,6 +226,21 @@ const Cover = ( {
 					style={ styles.rangeCellContainer }
 				/>
 			</PanelBody>
+
+			{ url ? (
+				<PanelBody title={ __( 'Media' ) }>
+					<BottomSheetButton
```

Comment:
For this case we could just import `BottomSheet` which exports pretty much all components that it uses included the `Button` something like

`import { BottomSheet } from '@wordpress/components';`

And then you can use `BottomSheet.Button` 

For this issue, I think we could go with `BottomSheet.Cell` 

Something like:

```
<BottomSheet.Cell
	leftAlign
	label={ __( 'Clear Media' ) }
	labelStyle={ styles.clearMedia }
	onPress={ () => {
		setAttributes( { id: undefined, url: undefined } );
	} }
/>
```

What do you think?